tests/config/test-custom.toml

```diff
@@ -16,6 +16,12 @@ end-point-stack-size = "12MB"
 end-point-recursion-limit = 100
 end-point-batch-row-limit = 64
 snap-max-write-bytes-per-sec = "10MB"
+readpool-read-critical-concurrency = 2
```

Comment:
do we have the critical priority command now?

Reply:
Critical priority is planned for operations like schema queries.

Reply:
if we don't use it now, remove it.